packages/cli/api-importers/openapi/openapi-ir-to-fern-tests/src/__test__/__snapshots__/openapi-v2/webhooks.json

```diff
@@ -154,7 +154,66 @@
   },
   "services": {},
   "errors": {},
-  "webhookGroups": {},
+  "webhookGroups": {
+    "": [
```

Comment:
@bilalmalik4321 this feels weird, the `""` if the operation id is empty i think we should default to something else as the id

Reply:
you could possibly generate something off of the id

Reply:
I used `operationId` in its place